po/pt_BR.po

```diff
@@ -46,15 +46,15 @@ msgstr "Todos os arquivos"
 
 #: src/main.js:83
 msgid "E-book Files"
-msgstr "Arquivos de e-book"
+msgstr "Arquivos de E-book"
 
 #: src/main.js:89
 msgid "Open File"
-msgstr "Abrir arquivo"
+msgstr "Abrir Arquivo"
 
 #: src/main.js:103
 msgid "translator-credits"
-msgstr ""
+msgstr "Créditos de tradutor"
```

Comment:
"translator-credits" should be the list of translators, separated by newlines, not the literal translation of "translator-credits".